pkg/supportbundle/defaultspec/spec.yaml

```diff
@@ -20,6 +20,18 @@ spec:
         selector:
           - app=kotsadm-postgres
         timeout: 10s
+    - exec:
+        collectorName: kotsadm-rqlite-db
+        name: kots/admin_console
+        containerName: rqlite
+        selector:
+          - app=kotsadm-rqlite
+        command:
+          - sh
+          - -c
+          - |
+            wget -qO- kotsadm:${RQLITE_PASSWORD}@localhost:4001/db/backup?fmt=sql  
```

Comment:
could we extract it from `/auth/config.json` somehow?  though this approach works and is probably cleaner.